src/com/task/constructions/CalculatorByIfElse.java

```diff
@@ -0,0 +1,27 @@
+package com.task.constructions;
+
+import java.util.Scanner;
+
+/**
+ * Умовна конструкція if - else.
+ * Створити клас Calculator і в ньому створити метод public void calculate (int number),
+ * який в залежності від того парна чи не парна цифра передана в нього буде виводити різні повідомлення у консоль.
+ */
+
+public class CalculatorByIfElse {
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int num = sc.nextInt();
+
+        calculate(num);
+    }
+
+     public static void calculate(int number) {
```

Comment:
you should create new CalculatoryByIfElse(); instance in main method, so you don't work with static variables, because it's not Object Oriented Programming approach, in OOP we're working with objects not with static. Static is for Procedural programming.